build.gradle

```diff
@@ -265,62 +265,47 @@ dependencyManagement {
     // CVE-2021-29425
     dependency group: 'commons-io', name: 'commons-io', version: '2.11.0'
 
-    dependencySet(group: 'org.springframework.cloud', version: '3.1.5') {
+    dependencySet(group: 'org.springframework.cloud', version: '4.0.2') {
```

Comment:
Do we need org.springframework.cloud, this has been removed in LAU

Reply:
Hi Martin , yes it is required and it's there in LAU as well.